### ISSUE TITLE
Update Project Profile: Access the Data

### ISSUE THREAD

**Issue by mxajPrice** [hackforla/website#4818]
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to update the project information for Access the Data so that visitors to the website can find accurate and up-to-date information.

#### Developer
- [x] In your IDE, find _projects/access-the-data.md file [^1] and make the following changes:
  - [x] For the `technologies` field, add the following:
    ```
      - Docker
      - Amazon Web Services (AWS)
      - PostgreSQL
    ```
  - [x] For the `tools` field,
    - [x] Remove Photoshop and Google Drive 
    - [x] Add GitHub, Zoom, and Google Suite
- [x] Using Docker, verify the changes by checking the Access the Data page [^2] in mobile, tablet, and desktop views

#### Merge Team
- [ ] Once the pr fixing this issue is merged, update issue #4798's Action Items section to reflect the updated `tools` data

### Resources/Instructions
- Original request copied into this comment for future reference: https://github.com/hackforla/website/issues/4818#issuecomment-1639108051
[^1]: Directory to find the page once you have it in your IDE: `_projects/access-the-data.md`
[^2]: Project page URL: https://www.hackforla.org/projects/access-the-data

**Comment by mxajPrice** [comment]
We also need to update our languages.

REMOVE:
- Jupyter Notebook

ADD:
- Python
- HTML
- CSS
- Typescript

**Comment by JessicaLucindaCheng** [comment]
# Original request

### Project Name

Access The Data

### Project Card URL

access-the-data.md

### Technologies

ADD:
- Docker
- Amazon Web Services (AWS)
- PostgreSQL

### Update the Tools?

YES

### Tools

REMOVE:
- Photoshop
- Google Drive

ADD:
- GitHub
- Zoom
- Google Suite


We also need to update our languages.

REMOVE:
- Jupyter Notebook

ADD:
- Python
- HTML
- CSS
- Typescript

**Comment by JessicaLucindaCheng** [comment]
@mxajPrice 

> We also need to update our languages.
> 
> REMOVE:
> 
> * Jupyter Notebook
> 
> ADD:
> 
> * Python
> * HTML
> * CSS
> * Typescript

The languages are pulled directly from the Access the Data GitHub repo: https://github.com/hackforla/access-the-data. Does Access the Data have another repo?

**Comment by mxajPrice** [comment]
@JessicaLucindaCheng ok, can we use our CKAN repo then for language tracking? 
https://github.com/hackforla/accessthedata-ckan

Access the Data is not the same project now as it started out as so there is some old code on the PM repo that is no longer being used. 

Thanks!

**Comment by JessicaLucindaCheng** [comment]
@mxajPrice 
Yes, the CKAN repo can be used: https://github.com/hackforla/accessthedata-ckan. However, it needs to have a `hack-for-la` topic tag. Are you able to add a `hack-for-la` topic tag to that repo?

**Comment by mxajPrice** [comment]
The hack-for-la topic tag is already there.
![Screen Shot 2023-07-25 at 14 14 19](https://github.com/hackforla/website/assets/98370780/a753aa0b-5383-47e0-a7f3-be1743ea7913)

**Comment by JessicaLucindaCheng** [comment]
@mxajPrice You're correct. Bonnie added it yesterday when I had a meeting with her. I'm sorry I did not update this issue sooner to let you know.

**Comment by JessicaLucindaCheng** [comment]
Notes:
- The repo id for https://github.com/hackforla/accessthedata-ckan is 508455581
- The `identification` will not be changed in the _projects/access-the-data.md file because there was no request to change the primary repo and Bonnie says it has to stay as the primary repo the project wants.
- The CTI repos have combined contributors because the current code adds up the contributions for each person in the `civictechindex` GitHub organization and uses that for the contributions for the CTI project. The current code only does this for projects outside the `hackforla` GitHub organization.

**Comment by davidwiese** [comment]
Availability: Today 7/31/23 2-4pm PST
ETA: Today 7/31/23 by 4pm PST

**Comment by davidwiese** [comment]
Progress Report:

    Progress: Complete
    Blockers: None
    Availability: N/A
    ETA: Complete

Before:
![Before](https://github.com/hackforla/website/assets/21321101/e1e94c37-22f1-4c1c-a94b-076d4bcf080b)

After:
![After](https://github.com/hackforla/website/assets/21321101/d8cde358-d363-4a62-8f1a-f51611bbbf22)

**Comment by JessicaLucindaCheng** [comment]
@Vinny02 Please do not move issues from the `New Issue Approval` column to the `Prioritized Backlog` because issues in the `New Issue Approval` column may not be ready to be worked on yet. Thank you.

**Comment by JessicaLucindaCheng** [comment]
@hackforla/website-merge I know this issue was moved when it wasn't ready yet and usually we take back issues moved that way. I spoke with Bonnie at the July 31, 2023 Dev/PM meeting about it and she approved the issue.

For this one issue, I think it is okay to let David continue working on the issue.

**Comment by JessicaLucindaCheng** [comment]
@mxajPrice Thank you for your patience. 

Since we don't manually update changes to languages and your project has more than one repo, this will be more complex and will be addressed in a separate issue.However, to not hold up all your requested changes, the Tools and Technologies

**Comment by Vinny02** [comment]
> @Vinny02 Please do not move issues from the `New Issue Approval` column to the `Prioritized Backlog` because issues in the `New Issue Approval` column may not be ready to be worked on yet. Thank you.

Woops, sorry about that Jessica, I don't know how that happened. I was not working on this issue, I might have moved it accidentally.

**Comment by JessicaLucindaCheng** [comment]
@mxajPrice Thank you for your patience. 

Since we don't manually update changes to languages and your project has more than one repo, this will be more complex and take more time. I have added your request for languages to be addressed in a separate issue: #5127 

However, to not hold up all your requested changes, the Tools and Technologies will still be changed as part of this issue.
